django_extensions/conf/model_class_template/model.py-tpl

```diff
@@ -0,0 +1,11 @@
+
+{% autoescape off %}
+class {{ model_name }}(models.Model):
```

Comment:
Add an option to change which parent class is being used from the command line ?

(With default being either ```models.Model``` or ```TimeStampedModel```)